collector/Makefile

```diff
@@ -17,19 +17,21 @@ COLLECTOR_BUILD_DEPS := $(HDRS) $(SRCS) $(shell find falcosecurity-libs -name '*
 
 .SUFFIXES:
 
-cmake-build/collector: $(COLLECTOR_BUILD_DEPS)
-	docker rm -fv build_collector || true
-	docker run --rm --platform ${PLATFORM} --name build_collector \
-		-v "$(BASE_PATH):$(SRC_MOUNT_DIR)" \
-		-e NPROCS=$(NPROCS) \
-		-e SRC_ROOT_DIR="$(SRC_MOUNT_DIR)" \
-		-e CMAKE_BUILD_TYPE=$(CMAKE_BUILD_TYPE) \
-		-e USE_VALGRIND=$(USE_VALGRIND) \
-		-e TRACE_SINSP_EVENTS=$(TRACE_SINSP_EVENTS) \
-		-e ADDRESS_SANITIZER=$(ADDRESS_SANITIZER) \
-		-e COLLECTOR_APPEND_CID=$(COLLECTOR_APPEND_CID) \
-		-e DISABLE_PROFILING="true" \
-		quay.io/stackrox-io/collector-builder:$(COLLECTOR_BUILDER_TAG) "$(SRC_MOUNT_DIR)/builder/build/build-collector.sh"
+cmake-configure/collector:
+	docker exec $(COLLECTOR_BUILDER_NAME) \
+		cmake -S $(BASE_PATH) -B $(CMAKE_DIR) \
+			-DCMAKE_BUILD_TYPE=$(CMAKE_BUILD_TYPE) \
+			-DDISABLE_PROFILING=ON \
```

Comment:
Wouldn't it be better to have it configurable via an env var as well?

Reply:
Yeah, I actually added this in #1393, but makes sense to have it here.